src/grokcore/component/compat.py

```diff
@@ -0,0 +1,31 @@
+import sys
+import martian
+from zope.interface import classImplements
+from zope.interface.interfaces import IInterface
+
+if sys.version_info[0] < 3:
+    from zope.interface import implements
+else:
+
+    class implements(martian.Directive):
+        scope = martian.CLASS
+        store = martian.ONCE
+        default = None
+        
+        def factory(self, *interfaces):
+            for interface in interfaces:
+                if not IInterface.providedBy(interface):
+                    raise Exception('%s is not a interface' % repr(interface))
+            return interfaces
+
+
+    class ClassImplementerGrokker(martian.ClassGrokker):
```

Comment:
Thanks, it's more clear to me know.

I see the build is now broken and the previous working build was last year.
If you get the build to pass, then I don't have any more objections. However, I'd be happy if someone else signs off as well.